Feat: Support multi language

Hekotoru opened the pull request:
### Feature

- Support multilingual.
- Rename `en` content folder to `es`.
- Create `es` menu configuration.

angelmadames commented on i18n/en.yaml:
```diff
@@ -1,17 +1,17 @@
 - id: get-started
-  translation: "¡Empezar!"
+  translation: "¡Get started!"
```
```suggestion
  translation: "Get started!"
```

angelmadames reviewed (changes requested):
Thank you so much for this! This sets the foundation for proper language tagging and future multilanguage features. You rock 🚀 

Left a minor typo fix suggestion. Also, I added some additional pages and fixed some links, thus probably why there are conflicts.

Hekotoru commented:
> Thank you so much for this! This sets the foundation for proper language tagging and future multilanguage features. You rock 🚀
> 
> Left a minor typo fix suggestion. Also, I added some additional pages and fixed some links, thus probably why there are conflicts.

A pleasure to contribute 😄 . All merge conflict and comments were fixed
